### ISSUE TITLE
Error log trouble 

### ISSUE THREAD

**Issue by criss20ster** [TeamFSIModding/FS17_Goweil_LT_Master#75]
Hello. I have such a problem, I use ultima and modu goweil. But when modularity is a problem, the problem disappears. but I would like to use both these fashions. Can you get this problem somehow?

errol log 
Error: Configuration Util.register Configuration Type too many configuration types. Only 64 configuration types are supported

**Comment by fcelsa** [comment]
Hi,
you probably have too many mods, so you have reached maximum number of configurations type supported by game.

**Comment by criss20ster** [comment]
How to check this configuration?

**Comment by fcelsa** [comment]
Check your log file, and search, almost at the beginning, the line like this:
`Farming Simulator 17 (Steam)
  Version: 1.5.3.1 b1848
  Available Languages: en de jp pl cz fr es ru it pt hu nl cs ct br tr ro kr
  Language: it
  Time: 2018-02-20 21:54:44
  Register configuration 'inputAttacherJoint'
  Register configuration 'attacherJoint'
  Register configuration 'frontloader'
  Register configuration 'motor'
  Register configuration 'baseColor'
  Register configuration 'wheel'
  Register configuration 'rimColor'
  Register configuration 'design'
  Register configuration 'designColor'
  Register configuration 'vehicleType'
Game vehicle types loaded`

probably you have a lot of mod that load it's own configuration, like goweil which register:
conveyorFlow
silageAdditiveSystem
dynamicChamber
monitorSystem
remoteMonitorSystem
starterKit

if you have many mods that register his configuration you can reach easily the maximum number allowed, 64, search in log and cut one or some of that register many config.

**Comment by criss20ster** [comment]
Ok I have a few Register configuration but how to find which exactly the mod corresponds to it
[log.txt](https://github.com/TeamFSIModding/FS17_Goweil_LT_Master/files/1745973/log.txt)

**Comment by Dzi4d3k** [comment]
@criss20ster 
Chodzi o to, że masz za dużo maszy rejestrujących własną konfigurację. Musisz coś wybrać bo przekraczasz limit narzucony przez grę. LT Master rejestruje 6 konfiguracji niestandardowych a inne mody z własnymi niestandardowymi konfiguracjami rejestrują swoje.

**Comment by criss20ster** [comment]
OK I understand. Only from among those modes that you see in the log as the game records which configuration is for which mode. So that I could choose a module or analyze it on which I care and on which I do not

**Comment by Dzi4d3k** [comment]
Dokładnie. Każdy mod z niestandardowymi skryptami powinien mieć plik register.lua a w nim takie linijki zaczynające się od
`ConfigurationUtil.registerConfigurationType("jakasKonfiguracja",...`
im więcej takich linijek tym więcej konfiguracji obsługuje mod.

@fcelsa 
Apologies for hacking this issue with Polish language, I think that @criss20ster I would explain issue more clearly in that way

**Comment by criss20ster** [comment]
Rozumiem. Z tego co zauważyłem w logu rejestruje się najpierw konfiguracja a pod spodem jest opis moda. I chyba tędy najlepsza droga do poszukania tych rejestrujących 

A słuchaj można jakoś połączyć jeden mod o nazwie Scania z 2 ? Bo mam taki mod gdzie jeden to scania 2- osiowa a 2 to 3-osiowa 

I ostatnie pytanie, gdzie moglibyśmy porozmawiać na spokojnie by nie zaburzać tutaj tego forum 

Dziękuje za pomoc i sugestie.

**Comment by Dzi4d3k** [comment]
Mi też tak to wygląda, że w logu listowane jest najpierw ładowanie konfiguracji a potem mod, ale to trzeba by sprawdzić porównując log z modami, których mogą wpisy z logu dotyczyć.

Co do łączenie dwóch modów w jedne, to nie pomogę za wiele, bo to nie jest takie proste, ale to nie temat tutaj. Załóż lepiej wątek na http://farming-simulator.pl, bo jak używasz tłumacza Google do pisania postów po angielsku to są kiepskie efekty za tego.

**Comment by criss20ster** [comment]
Wiem. Zauważyłem sorrki. Ale chciałem by ktoś mi jakoś pomógł, wspomógł w problemie i nakierował. Ale ok wiem że cała społeczność opiera się na angielskim. Ale ok już mam pomysł na te konfiguracje ;)
